packages/trader/src/Modules/Trading/Containers/trade.tsx

```diff
@@ -114,6 +115,10 @@ const Trade = observer(() => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [symbol, setDigits, setTrySyntheticIndices, is_synthetics_available]);
 
+    React.useEffect(() => {
+        setOpenLaunchModal(is_turbos);
```

Comment:
As Turbos now become a default trade type for regions, where it is available, I decided to use `is_turbos` for check (show modal or not)

Reply:
Will it not trigger modal multiple times when users switch between different trade types?

Reply:
No, it won't because on line 163 there is a condition for opening: `open_launch_modal && is_logged_in && !is_already_shown `
So, if user been already shown this pop-up, then he win't be able to see it again (until he clean up his localStorage)